### PR TITLE
Add Phone assert

### PR DESCRIPTION
**Pull request by promag**
R=@nunofgs

### CODE REVIEW

File: test/asserts/country-phone-assert_test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const Assert = BaseAssert.extend({
+  CountryPhone: CountryPhoneAssert
+});
+
+/**
+ * Test `CountryPhoneAssert`.
+ */
+
+describe('CountryPhoneAssert', () => {
```
Comment (promag) [inline test/asserts/country-phone-assert_test.js]: Missing tests for constructor.

File: src/asserts/country-phone-assert.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   * Optional peer dependencies.
+   */
+
+  const countries = require('isoc');
+
+  /**
+   * Class name.
+   */
+
+  this.__class__ = 'CountryPhone';
```
Comment (nunofgs) [inline src/asserts/country-phone-assert.js]: Rename to `PossiblePhone`?

Comment (promag) [inline src/asserts/country-phone-assert.js]: :+1:

File: src/asserts/possible-phone-assert.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  const countries = require('isoc');
+
+  /**
+   * Class name.
+   */
+
+  this.__class__ = 'PossiblePhone';
+
+  // Test by `alpha-3` code.
+  let country = find(countries, { alpha3: code });
```
Comment (nunofgs) [inline src/asserts/possible-phone-assert.js]: Still debating whether we should accept _alpha3_ as well as _alpha2_. Maybe we should just accept _alpha2_ for simplicity's sake?

WDYT @ruimarinho?

File: src/asserts/possible-phone-assert.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+  // Test by `alpha-2` code.
+  if (!country) {
+    country = find(countries, { alpha2: code });
+  }
+
+  if (!country) {
+    throw new Error('A valid ISO-3166 country code must be specified');
```
Comment (nunofgs) [inline src/asserts/possible-phone-assert.js]: If we only accept _alpha2_, we might not even need to perform this check. We could simply pass it to `phoneUtil.getRegionCodeForNumber()` if it already does error handling.

File: src/asserts/possible-phone-assert.js (hunk 3 above)
Comment (promag) [inline src/asserts/possible-phone-assert.js]: No problem for me.

**Pull request merged**